ecosystem/resources/members/povm-toolbox.toml

```diff
@@ -0,0 +1,8 @@
+name = "povm-toolbox"
```

Comment:
```suggestion
name = "POVM Toolbox"
```

@mrossinek is this ok? We're trying to avoid titles being slugs where possible

Reply:
Yes sure! :+1:

Reply:
Thanks! Also, any ideas what the best category is (we haven't added that form to the issue template yet)? Options are:

* Transpiler pass
* System providers
* Applications packages
* Other